### DISCUSSION TITLE
Issues installing security onion.

### DISCUSSION THREAD

**Discussion by wuljungba** [Security-Onion-Solutions/securityonion#6987] [Unsupported Versions] [answered]
![Screenshot (270)](https://user-images.githubusercontent.com/63914988/151085129-17279bd7-4ebd-49ec-8aec-d88171aa6c11.png)
Please see screenshot,not sure why so-preflight keeps failing each time i try installing security onion on both eval or standelone .Any help will be great.I am using vmware workstation.Also my vmnet default gateway ip's are blank when i run ipconfig.Can that be the problem and how do i resolve the problem.Not sure how to access root /sosetup  if install not complete 
Thank
![Screenshot (270)](https://user-images.githubusercontent.com/63914988/151085872-93f3072d-9499-4c28-a97a-986acf8dc2d1.png)
s

**Comment by nutt318** [comment]
I had this same issue and it was the firewall blocking a few sites, *.docker.com, *saltstack.com, *.wazuh.com had to be added to a whitelist policy.

**Comment by dougburks** [comment, marked as answer]
> vmnet default gateway ip's are blank when i run ipconfig.Can that be the problem

Yes, that's a problem. Please perform a fresh installation, follow the screenshots [here](https://docs.securityonion.net/en/2.3/first-time-users.html), and make sure you that you specify a gateway:

![image](https://user-images.githubusercontent.com/1659467/151360547-356cca68-34b8-4a4b-99bc-9503a6362e72.png)
